src/NUnitFramework/framework/nunit.framework-2.0.csproj

```diff
@@ -436,7 +436,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="NUnit.System.Linq, Version=0.2.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\NUnit.System.Linq.0.2.0-CI-12-master\lib\net20\NUnit.System.Linq.dll</HintPath>
+      <HintPath>..\..\..\packages\NUnit.System.Linq.0.2.0-CI-17-master\lib\net20\NUnit.System.Linq.dll</HintPath>
```

Comment:
I did the same change in the release 3.4 branch, so you will get a merge conflict. I would prefer if we merge my 3.4 PR before this one so that this one will get labeled 3.5 once we merge.